### PR TITLE
Move wiki to docs

### PR DESCRIPTION
**Pull request by godwhoa**
Also would appreciate some help moving more pages.

- [x] Docker
- [x] CLI 
- [x] Configuration
- [x] Porting scripts
- [x] Contributing (user)
- [x] Installation
- [x] Contributing (developer)
- [x] Home (updated version)

Closes: #317

### CODE REVIEW

**Comment by feikname** [comment]
I will attempt to help with the move later tonight.

**Comment by feikname** [comment]
So many typos fixed 😆

File: doc/configuration.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Configuring Piqueserver
+\Configuring Piqueserver
```
Comment (feikname) [inline doc/configuration.rst]: Why the `\` at the beggining here?

Comment (godwhoa) [reply doc/configuration.rst]: Whoops. That shouldn't be there.

**Comment by godwhoa** [comment]
Maybe we should merge development.rst and contributing-developer.rst?

**Comment by feikname** [comment]
Also, rename the `docs & wiki` label to only `docs` once this gets merged.

File: doc/docker.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    # build it locally
+    git clone https://github.com/piqueserver/piqueserver
+    cd piqueserver
+    docker build -t piqueserver/piqueserver:master .
+
+
+Running
+-------
+
+Get your config. ready. It should have the following structure:
```
Comment (swalladge) [inline doc/docker.rst]:
```suggestion
Get your config ready. It should have the following structure:
```

typo?

**Comment by feikname** [comment]
@swalladge One is for developers and the other for common users of piqueserver.

The reason one of them is named `contributing.rst` is so GitHub can automatically pick it up (https://help.github.com/articles/setting-guidelines-for-repository-contributors/)

Now that I think about it, it makes more sense for the user guide to be `contributing.rst` rather than the developer guide, IHMO.

**Comment by godwhoa** [comment]
"Config directory structure" should probably be in "Configuring Piqueserver" and also updated.

**Pull request merged**